### PR TITLE
feat: envs

### PR DESCRIPTION
**Pull request by infantesimone**

### CODE REVIEW

**Review by giovanniberti — COMMENTED** [review]
test comment

File: global-transaction-service.env [hunk 1]
```diff
@@ -0,0 +1,28 @@
+MONGO_PORT=27017
```
Comment (giovanniberti) [inline global-transaction-service.env:1]: Is this file used anywhere?

Comment (giovanniberti) [reply global-transaction-service.env:1]: Here: https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/4dad79fa35a56c6932b13b51ca68d4ae22823eb2/.devops/code-review-pipelines.yml#L81

File: .env [hunk 2]
```diff
@@ -4,7 +4,7 @@ MONGO_EXPRESS_PORT=8081
 REDIS_PORT=6379
 REDIS_INSIGHT_PORT=8001
 
-NODO_MOCK_COMMIT_SHA=main
+NODO_MOCK_COMMIT_SHA=fix-namespace-verifica
```
Comment (Fernando-Granato) [inline .env:7]: Is it right to leave this branch as a reference ?

**Review by Fernando-Granato — APPROVED** [review]

File: .env (hunk 2 above)
Comment (infantesimone) [reply .env:7]: yes, temporary fix -> https://github.com/pagopa/io-pagopa-node-mock/pull/40

**Pull request merged**